src/components/App/App.js

```diff
@@ -0,0 +1,28 @@
+require('normalize.css/normalize.css')
+require('./App.css')
+
+import React from 'react'
+import Navbar from '../Navbar/Navbar'
+import { Switch, Route } from 'react-router-dom'
+import Shop from '../../screens/Shop/Shop'
```

Comment:
I met some problem - I want divide components into presentational and containers (https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0) but now `containers` folder name can be confusing. I don't think to put there all my containers is a good idea. I want to keep there on the top only my 'views' so maybe is better stay with `scenes` or `screens` naming?